cmake/FindCPPHOCON.cmake

```diff
@@ -0,0 +1,6 @@
+include(FindDependency)
```

Comment:
We should set up cpp-hocon itself to do the normal CMake library export soon-ish, but since Facter is the only consumer I think we can fix that when we switch cpp-hocon to be shared.